typescript/src/openapi-client/index.ts

```diff
@@ -57,3 +57,4 @@ export const OpenApiPoliciesMethods = {
 };
 
 export type CdpOpenApiClientType = typeof CdpOpenApiClient;
+export * from "./generated/webhooks/webhooks.js";
```

Comment:
based on the other exports in this file, i think this should be `export * as webhooks from "./generated/webhooks/webhooks.js";`, and it should move up to the top of the file

Reply:
yeah good call.